src/module.c

```diff
@@ -7672,13 +7671,13 @@ void RM_LogIOError(RedisModuleIO *io, const char *levelstr, const char *fmt, ...
     va_end(ap);
 }
 
-/* Redis-like assert function.
+/* serverAssert-like function.
```

Comment:
`redisAssert` is an internal function. It's not familiar to module API users. Better something like this...

```suggestion
/* Valkey assert function.
```